script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -382,13 +388,14 @@ But what's with the change of heart? Wasn't Vlov your only enemy?"
 -- Page 461, Offset 15901.
 -- 「？　わたしの敵はヴローヴじゃないよ？
 // ？ is voiced
-"Huh? My enemy wasn't Vlov though.
+"Huh? Vlov wasn't my enemy.
 }
 [sha:ad016f3eff1d767ca1716d36a98e40300e207c51]{
 -- Page 461, Offset 15902.
 -- わたしが探していたのはこの街に巣くった吸血鬼で、ヴローヴは関係ないもの」
 // from here on, much of the conversation mirrors stuff from ARC07_3D
-The vampire I was looking for lurks in this city――Vlov was unrelated"
+I was looking for a different vampire that lurked in this city,
```

Comment:
Revert; Vlov didn't actually lurk in the city.

Reply:
I mean, he might not have lurked for as long as Roa, but he _did_ have his lair which I'd say is worthy of the term "lurking"

Reply:
No, Vlov didn't actually have a lair.

This is misdirection from Nasu.

Reply:
I feel like lurking is pretty general 'up to no good' vibes, you don't necessarily have to set up shop.